# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/moz.build
# Commit: 432c7e9dc19f
# Full Hash: 432c7e9dc19fffeb05f470f6eb7bc9f619c82a2d
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-05 21:44:54
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P1: Introduce nsWebSocketConnection interface for separating socket manipulation r=michal,necko-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30023
# ==============================================================================

diff -r 0d22ad297b19 -r 432c7e9dc19f netwerk/protocol/websocket/moz.build
--- a/netwerk/protocol/websocket/moz.build	Wed Aug 05 16:44:36 2020 +0000
+++ b/netwerk/protocol/websocket/moz.build	Mon Aug 03 10:03:08 2020 +0000
@@ -10,6 +10,7 @@
 XPIDL_SOURCES += [
     'nsITransportProvider.idl',
     'nsIWebSocketChannel.idl',
+    'nsIWebSocketConnection.idl',
     'nsIWebSocketEventService.idl',
     'nsIWebSocketImpl.idl',
     'nsIWebSocketListener.idl',
@@ -32,6 +33,7 @@
 UNIFIED_SOURCES += [
     'BaseWebSocketChannel.cpp',
     'IPCTransportProvider.cpp',
+    'nsWebSocketConnection.cpp',
     'WebSocketChannel.cpp',
     'WebSocketChannelChild.cpp',
     'WebSocketChannelParent.cpp',